libs/portal-integration-angular/src/lib/core/components/data-view-controls/data-view-controls.component.html

```diff
@@ -77,6 +83,7 @@
       offIcon="pi pi-sort-amount-down"
       (onChange)="sortDirection($event)"
       [title]="!selectedSortDirection ? (translations?.sortOrderTooltips?.ascending || ('OCX_LIST_GRID_SORT.TOGGLE_BUTTON.ASCENDING_TITLE' | translate)) : (translations?.sortOrderTooltips?.descending || ('OCX_LIST_GRID_SORT.TOGGLE_BUTTON.DESCENDING_TITLE' | translate))"
+      [ariaLabel]="!selectedSortDirection ? (translations?.sortOrderTooltips?.ascending || ('OCX_LIST_GRID_SORT.TOGGLE_BUTTON.ASCENDING_TITLE' | translate)) : (translations?.sortOrderTooltips?.descending || ('OCX_LIST_GRID_SORT.TOGGLE_BUTTON.DESCENDING_TITLE' | translate))"
```

Comment:
[attr.aria-label] ?

Reply:
p-toggleButton has `ariaLabel` prop which should be used in this case